### ISSUE TITLE
Crash on practice cat customer fail

### ISSUE THREAD

**Issue by zokradonh** [KitchenMods/KitchenLib#48]
**Describe the bug**
Crash on failing to satisfy the practice cat customers

**To Reproduce**
1. Install Harmony and KitchenLib mod by Steam Workshop
2. Start PlateUp
3. Enter the practice kitchen in the "lobby" (you do not need to be in a run) and trigger the cats
4. fail to satisfy them -> crash

**Expected behavior**
No crash

**Other Installed Mods**
- Harmony

**Additional context**
I have deinstalled all mods to track down the problem until uninstalling KitchenLib fixed the problem. Only Harmony was installed at that time. I have reinstalled only KitchenLib and the problem was present again.

**Comment by zokradonh** [comment]
```
NullReferenceException: Object reference not set to an instance of an object
  at KitchenLib.RemoveBlacklistedSides.OnUpdate () [0x00074] in <198fb284a4c4424286622bdb5ebc9dc1>:0 
  at Unity.Entities.SystemBase.Update () [0x0007c] in <db13783bcc9e496f8dfc62742309ca8d>:0 
  at Unity.Entities.ComponentSystemGroup.UpdateAllSystems () [0x000e0] in <db13783bcc9e496f8dfc62742309ca8d>:0 
UnityEngine.DebugLogHandler:Internal_LogException(Exception, Object)
UnityEngine.DebugLogHandler:LogException(Exception, Object)
UnityEngine.Logger:LogException(Exception, Object)
UnityEngine.Debug:LogException(Exception)
Unity.Debug:LogException(Exception)
Unity.Entities.ComponentSystemGroup:UpdateAllSystems()
Unity.Entities.ComponentSystemGroup:OnUpdate()
Unity.Entities.ComponentSystem:Update()
Unity.Entities.DummyDelegateWrapper:TriggerUpdate()
```

**Comment by zokradonh** [comment]
It seems it only happens when this line is present:

https://github.com/KitchenMods/KitchenLib/blob/006097c8b562c25b94eaa9cb2fff4ed55e52fa40/KitchenLib/src/Systems/RemoveBlacklistedSides.cs#L22

Due to the lack of my Unity experience I can not help here.
Until I realized that Unity can throw `NullReferenceException`s for structs I wasted already some time.

Thank you for your work

**Comment by zokradonh** [comment]
Fixed in 0.5.9
